libbeat/docs/monitoring/monitoring-beats.asciidoc

```diff
@@ -0,0 +1,69 @@
+//////////////////////////////////////////////////////////////////////////
+//// This content is shared by all Elastic Beats. Make sure you keep the
+//// descriptions here generic enough to work for all Beats that include
+//// this file. When using cross references, make sure that the cross
+//// references resolve correctly for any files that include this one.
+//// Use the appropriate variables defined in the index.asciidoc file to
+//// resolve Beat names: beatname_uc and beatname_lc.
+//// Use the following include to pull this content into a doc file:
+//// include::../../libbeat/docs/monitoring/configuring.asciidoc[]
+//// Make sure this content appears below a level 2 heading.
+//////////////////////////////////////////////////////////////////////////
+
+[role="xpack"]
+[[monitoring]]
+= Monitoring {beatname_uc}
+
+[partintro]
+--
+{monitoring} enables you to easily monitor {beatname_uc} from {kib}. For more
+information, see
+{xpack-ref}/xpack-monitoring.html[Monitoring the Elastic Stack] and
+{kibana-ref}/beats-page.html[Beats Monitoring Metrics].
+
+To configure {beatname_uc} to collect and send monitoring metrics:
+
+. Create a user that has appropriate authority to send system-level monitoring
+data to {es}. For example, you can use the built-in `logstash_system` user or
+assign the built-in `logstash_system` role to another user. For more
+information, see
+{xpack-ref}/setting-up-authentication.html[Setting Up User Authentication] and
+{xpack-ref}/built-in-roles.html[Built-in Roles].
```

Comment:
It should be clear the generic user is not named `beats_system`, as this name will be used in future revisions.